src/game/game.c

```diff
@@ -13,6 +13,17 @@
 
 #include <SDL2/SDL_image.h>
 
+static void set_window_title(struct oshu_game **game)
+{
+	const char * title = "Oshu! ♫ \"%s\" by %s";
+
+	asprintf(&(*game)->display->window_title, title,
+		(*game)->beatmap->metadata.title,
+		(*game)->beatmap->metadata.artist);
+	SDL_SetWindowTitle((*game)->display->window,
+		(*game)->display->window_title);
```

Comment:
Please align the arguments.

Either move the first argument a separate line and use tabs, like I often do, or use spaces to align, but don't assume they're 8-chars long. `asprintf(` takes 9 chars, so that will be 9 spaces. For reference, look at the source code of Linux, I know you have it at `/usr/src/linux`.